tests/ludwig/features/test_binary_feature.py

```diff
@@ -0,0 +1,26 @@
+from typing import Dict
+
+import pytest
+import torch
+
+from ludwig.features.binary_feature import BinaryInputFeature
+
+SEQ_SIZE = 2
+BINARY_W_SIZE = 1
+
+
+@pytest.fixture(scope="module")
+def binary_config():
```

Comment:
Not sure if one exists, but it would be great if there was a schema we could point to as a comment.